uwv_dynamic_model.orogen

```diff
@@ -121,6 +121,6 @@ task_context "VelocityEstimator" do
         max_latency 0.05
     end
 
-    runtime_states :MISSING_TRANSFORMATION
+    exception_states :MISSING_TRANSFORMATION
```

Comment:
You cannot make MISSING_TRANSFORMATION an exception state. With the way the transformer is currently implemented, it is expected in the few first samples, as the transformer "discovers" the transformation streams.